src/styles/blocks/header.scss

```diff
@@ -0,0 +1,55 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  box-shadow: $header-shadow;
+
+  &__logo {
+    margin: 10px 0 0 50px;
```

Comment:
U can use a display block and mixin square for the logo ;)

Reply:
i'm not sure this is necessary, cuz `img` already has direct size and im'not going to change it